.github/actions/release/macos/action.yml

```diff
@@ -0,0 +1,42 @@
+name: Release for macos
+inputs:
+   tag:
+      description: Tag
+      required: true
+      type: string
+   GH_TOKEN:
+      description: GitHub token
+      required: true
+      type: string
+runs:
+   using: composite
+   steps:
+      -  name: Setup Go
+         uses: actions/setup-go@v5
+         with:
+            go-version-file: "go.mod"
+
+      -  shell: bash
+         run: brew install xmlstarlet
+
+      -  shell: bash
+         run: brew install create-dmg
+
+      -  shell: bash
+         run: mkdir dist
```

Comment:
ditto. Use one step for installing deps and move making `dist` folder to build step